### PR TITLE
[json-dto] Update to 0.2.12

### PR DESCRIPTION
**Pull request by eao197**
json_dto library updated to the latest version.

### CODE REVIEW

File: ports/json-dto/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "json-dto",
-  "version-string": "0.2.11",
+  "version-string": "0.2.12",
```
Comment (JonLiu1993) [inline ports/json-dto/vcpkg.json:3]: @eao197 ,Thanks for your pr, please change the "version-string" to "version".

File: versions/j-/json-dto.json [hunk 2]
```diff
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "3231926e8d2e3ce9237b55c597c24baa344134a0",
+      "version-string": "0.2.12",
```
Comment (JonLiu1993) [inline versions/j-/json-dto.json:5]: please change the "version-string" to "version".

File: ports/json-dto/vcpkg.json (hunk 1 above)
Comment (eao197) [reply ports/json-dto/vcpkg.json]: Done

File: versions/j-/json-dto.json (hunk 2 above)
Comment (eao197) [reply versions/j-/json-dto.json]: Done.

**Review by JonLiu1993 — APPROVED** [review]

**Pull request merged**